docs/introduction_tutorial/intro_tutorial_part3.ipynb

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
<p>Recipe is not actually defined here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>